securedrop/models.py

```diff
@@ -35,6 +35,10 @@
 ARGON2_PARAMS = dict(memory_cost=2**16, rounds=4, parallelism=2)
 
 
+def generate_otp_secret() -> str:
+    return base64.b32encode(os.urandom(20)).decode()
```

Comment:
Since the move to Focal SD cand probably use secrets (Python 3.6+) now, so you could also do something like:

```
import string
import secrets

def generate_otp_secret() -> str:
    symbols = string.ascii_uppercase + string.digits
    return ''.join(secrets.choice(symbols) for i in range(OTP_LENGTH))
```

Might be good to abstract away from os.urandom, and it achieves the goal of taking pyotp out of the requirements.

Reply:
I agree actually. would you like to author this ?